test/built-ins/RegExp/prototype/Symbol.matchAll/get-species-constructor-err.js

```diff
@@ -1,22 +0,0 @@
-// Copyright (C) 2015 the V8 project authors. All rights reserved.
```

Comment:
This test is now: [species-constructor-get-species-throws.js](https://github.com/peterwmwong/test262/blob/matchall-pww/test/built-ins/RegExp/prototype/Symbol.matchAll/species-constructor-get-species-throws.js)